app/models/financial_transaction.rb

```diff
@@ -18,6 +18,14 @@ class FinancialTransaction < ActiveRecord::Base
     initialize_financial_transaction_type
   end
 
+  def self.ransackable_attributes(auth_object = nil)
+    %w(id amount note created_on user_id)
```

Comment:
here is as `created_on` too

Reply:
Thanks! I've updated this. Not too pretty, but I'd rather to #575 separately.